diff --git a/js/paragraphs_ee.admin.js b/js/paragraphs_ee.admin.js
index e970380..d6a7bc9 100644
--- a/js/paragraphs_ee.admin.js
+++ b/js/paragraphs_ee.admin.js
@@ -81,8 +81,7 @@
       var rowConfig = {
         buttons: buttonList,
         buttonCount: buttonCount,
-        text: Drupal.t('...', {}, {context: 'Paragraphs Editor Enhancements'}),
-        title: title,
+        text: title,
         dialog: $addModalDialog
       };
       if (buttonsMax === 0) {
